src/interpreter.cpp

```diff
@@ -1686,9 +1688,15 @@ void command_interpreter(struct char_data * ch, char *argument, const char *tcna
         break;
       case POS_MORTALLYW:
         send_to_char("You are in a pretty bad shape! You can either wait for help, or give up by typing ^WDIE^n.\r\n", ch);
+        if (PLR_FLAGGED(ch, PLR_NEWBIE)) {
+          send_to_char(ch, "^L[OOC: While your TKE is less than 50, your only penalty for dying is losing your current job. See ^wHELP NEWBIE^L for more details.]\r\n");
```

Comment:
For messages where there are no format specifiers in the output, please use the form of `send_to_char("message", ch);` as this avoids putting the string through printf entirely. This is good practice to be in so we don't inadvertently introduce memory leaks / crash conditions if we write messages that have unguarded percent signs in them.

Reply:
Got it.